src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -153,37 +170,124 @@ object IndexCollectionManager {
 }
 
 /**
- * Case class representing index summary
- *
- * TODO: Finalize about adding these: data location, signatures, file lists etc.
+ * Case class representing index metadata and index statistics from latest index version.
```

Comment:
Can you move `IndexStatistics` related code to `IndexStatistics.scala` (not to pollute `IndexCollectionManager.scala`)? You can also move `getIndexContentDirectoryPaths` and `indexDirPath` to `object IndexStatistics`.